libfirm-rs/generator/gen.ts

```diff
@@ -0,0 +1,269 @@
+import fs from "fs";
```

Comment:
Why typescript? :sob: This could be nicely done with serde-json and `proc_macro`s...